content/pipeline/exec/syntax/trigger.md

```diff
@@ -294,4 +294,51 @@ trigger:
   target:
     exclude:
     - production
-{{< / highlight >}}
\ No newline at end of file
+{{< / highlight >}}
+
+# By Action
+
+The action trigger limits execution based on the action associated with the event that triggered the pipeline.
+
+<div class="alert">
+Action support varies by event and SCM provider.
+</div>
+
+{{< highlight text "linenos=table,linenostart=12" >}}
+trigger:
+  action:
+  - labeled
+{{< / highlight >}}
+
+Execute when a pull request is opened or reopened:
+
+{{< highlight text "linenos=table,linenostart=12" >}}
+trigger:
+  event:
+  - pull_request
+  action:
+  - opened
+  - reopened
+{{< / highlight >}}
+
+Execute on all pull request actions except "synchronize":
+
+{{< highlight text "linenos=table,linenostart=12" >}}
+trigger:
+  event:
+  - pull_request
+  action:
+    exclude:
+    - synchronize
```

Comment:
I believe this should be `syncronized` (with a `d` at the end)
https://github.com/drone/go-scm/blob/master/scm/const.go#L65